source/getting-started/onboarding.markdown

```diff
@@ -3,56 +3,53 @@ title: "Onboarding Home Assistant"
 description: "Instructions to get Home Assistant configured."
 ---
 
-Alright, you made it here. The tough part is done.
+After your Home Assistant server is up and running, there are 5 simple steps to log in and see your devices.
 
-- As the last step of the [installation procedure](/installation/) specific to your device, you entered the Home Assistant device's address in your browser's address bar.
-    - Typically, this is `http://homeassistant.local:8123/`.
-    - If you are running an older Windows version or have a stricter network configuration, you might need to access Home Assistant at <a href="http://homeassistant:8123" target="_blank">homeassistant:8123</a> or `http://X.X.X.X:8123` (replace X.X.X.X with your device's IP address).
+**Step 1**: type  <http://homeassistant.local:8123/> into your browser.
```

Comment:
I'm not a fan of using a bold style emphasis instead of headers.

Headers are important for search engines to extract content and know the importance, including or own search engine.